console/views/FTP.py

```diff
@@ -44,12 +29,20 @@ def get_file_contents(datatype, filename):
 @FTP_bp.route('/FTP', methods=['POST', 'GET'])
 @flask_security.login_required
 def ftp_home():
-    return render_template('FTP.html', enable_empty_ftp=settings.ENABLE_EMPTY_FTP)
+    with ConsoleFtp() as ftp:
+        contents = ftp.get_folder_contents(PATHS["pck"])[0:20]
+    return render_template('FTP.html', enable_empty_ftp=settings.ENABLE_EMPTY_FTP, contents=contents)
+
 
+@FTP_bp.route('/FTP/<datatype>', methods=['POST', 'GET'])
+@flask_security.login_required
+def ftp_pcks(datatype):
+    if datatype not in ['pck', 'image', 'index', 'receipt', 'json']:
+        return render_template('FTP.html', enable_empty_ftp=settings.ENABLE_EMPTY_FTP)
 
-@FTP_bp.route('/ftp.json')
-def ftp_list():
-    return jsonify(get_ftp_contents())
+    with ConsoleFtp() as ftp:
+        contents = ftp.get_folder_contents(PATHS[datatype])[0:20]
+    return render_template('FTP.html', enable_empty_ftp=settings.ENABLE_EMPTY_FTP, contents=contents)
```

Comment:
Most of this is re-done, would it be worth making it one, or not loading the data until we click a tab?

Reply:
umm, what do you mean sorry?  I've made it so that if you click on the FTP tab, it loads the pck data.  And if you click on any of the sub tabs, it'll load that data instead.

Reply:
Yeah, so that loads pck throug 'ftp_home' above this route, would it be worth loading the ftp page without the data? Just a thought more than anything.